code/weapon/weapons.cpp

```diff
@@ -1333,6 +1333,14 @@ int parse_weapon(int subtype, bool replace, const char *filename)
 				stuff_float(&wip->turn_time);
 			}
 
+			if (optional_string("+Turning Acceleration Time:")) {
+				stuff_float(&wip->turn_accel_time);
+				if (!Framerate_independent_turning) {
+					Warning(LOCATION, "Turning Acceleration Time for weapon %s requires \"AI use framerate independent turning\" in game_settings.tbl ", wip->name);
```

Comment:
Since this would be true for any entry where they attempt to use `+Turning Acceleration Time:`, and they only need to set `Framerate_independent_turning` once to fix it for all the entries that use this new flag, we should probably use a static bool to only show this warning once.